lib/builder_web.ml

```diff
@@ -639,6 +708,7 @@ let routes ~datadir ~cachedir ~configdir ~expired_jobs =
     `Get, "/all-builds", (w (builds ~all:true));
     `Get, "/hash", (w hash);
     `Get, "/compare/:build_left/:build_right", (w compare_builds);
+    `Get, "/compare/json/:build_left/:build_right", (w compare_builds_json);
```

Comment:
the alternative, that Reynir suggested, was to use the same endpoint, and respect the `Accept` header (whether it contains `application/json` or not.